args_test.go

```diff
@@ -293,6 +471,32 @@ func TestMatchAll(t *testing.T) {
 	}
 }
 
+// DEPRECATED
+
+func TestExactValidArgs(t *testing.T) {
+	c := getCommand(ExactValidArgs(3), true)
+	output, err := executeCommand(c, "three", "one", "two")
+	expectSuccess(output, err, t)
+}
+
+func TestExactValidArgs_WithInvalidCount(t *testing.T) {
+	c := getCommand(ExactValidArgs(2), false)
+	_, err := executeCommand(c, "three", "one", "two")
+	exactArgsWithInvalidCount(err, t)
+}
+
+func TestExactValidArgs_WithInvalidCount_WithInvalidArgs(t *testing.T) {
+	c := getCommand(ExactValidArgs(3), true)
```

Comment:
Based on the test title, I believe we want to specify the wrong number of args and the wrong values for them. 
 So, we should probably specify 2 args here since we want the next line to be wrong and it specifies 3 args

Reply:
Done in https://github.com/spf13/cobra/pull/1643/commits/4e31482378d51181ed0651cd6daf6a6aa2e946ab.